src/python/pants/option/parser.py

```diff
@@ -456,12 +456,13 @@ def to_value_type(val_str):
 
     # Helper function to expand a fromfile=True value string, if needed.
     # May return a string or a dict/list decoded from a json/yaml file.
-    def expand(val_str):
-      if kwargs.get('fromfile', False) and val_str and val_str.startswith('@'):
-        if val_str.startswith('@@'):   # Support a literal @ for fromfile values via @@.
-          return val_str[1:]
+    def expand(val_or_str):
+      if (kwargs.get('fromfile', True) and val_or_str and
```

Comment:
Does `kwargs` ever contain `'fromfile'` here or is this using the default 100% of the time? If the latter, drop the clause?

Reply:
We haven't deprecated `fromfile` yet, so it could in theory be set in an external plugin.